tests/drivers/can/api/src/canfd.c

```diff
@@ -257,22 +257,20 @@ ZTEST(canfd, test_send_fd_dlc_out_of_range)
 }
 
 /**
- * @brief Test error when CAN FD Error State Indicator (ESI) is send without FD format flag (FDF).
+ * @brief Test error when CAN FD Error State Indicator (ESI) is set on transmit frame.
  *
- * CAN FD Error State Indicator (ESI) indicates that the transmitting node is
- * in error-passive state. Only valid in combination with CAN_FRAME_FDF.
```

Comment:
This was copied from the CAN API documentation at
https://docs.zephyrproject.org/latest/doxygen/html/group__can__interface.html#ga83f8b7af6eb45e43aaac355de3e69e52
Is it worth to make changes there too?